pylint/utils/normalize_text.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+import textwrap
+
+
+def normalize_text(text, line_len=80, indent=""):
```

Comment:
Seems odd to have a module just for this function, is there a way to move it somewhere else?

Reply:
It's used in a lot of places and I think there is a circular import problem (Like for WarningScope). It's probably possible to fix, but that require some design.